src/test/java/eu/bbmri_eric/negotiator/integration/service/UserNotificationServiceTest.java

```diff
@@ -278,4 +273,60 @@ void notifyUsersForNewPost_createdByRep_requesterIsNotified() {
     assertFalse(
         notificationRepository.findByRecipientId(negotiation.getCreatedBy().getId()).isEmpty());
   }
+
+  @Test
+  @WithMockNegotiatorUser(id = 109L, roles = "ADMIN")
+  public void testSendReminderEmails() {
```

Comment:
Not sure one test is enough for this. You should check multiple scenarios: for example set trigger duration to 0ms and then check if correct notifications were made